pints/tests/test_toy_gaussian_logpdf.py

```diff
@@ -132,6 +137,24 @@ def test_gaussian_sensitivity(self):
         self.assertAlmostEqual(L, -25.10903637045394)
         self.assertTrue(np.array_equal(dL, [0.25, 3.5, 3.0]))
 
+        # check incorrect covariance matrices being added
+        self.assertRaises(ValueError,
+                          pints.toy.GaussianLogPDF, [1, 2], [[1, 0.5],
+                                                             [-0.5, 2]])
+        self.assertRaises(ValueError,
```

Comment:
It's a personal preference, but I'd have written these as
```
self.assertRaises(
    ValueError, ...)
```
Just out of interested, do you find the current form prettier / easier to read?